docs/csharp/programming-guide/concepts/serialization/walkthrough-persisting-an-object-in-visual-studio.md

```diff
@@ -23,6 +23,11 @@ In this walkthrough, you will create a basic `Loan` object and persist its data
 > [!TIP]
 > Need to install a code editor? Try [Visual Studio](https://visualstudio.com/downloads)!
 
+* The below code example requires C# 7.3 to be selected as the language version. Please refer to below 
```

Comment:
- The below example code requires C# 7.3 to be selected as the language version. --> The example requires C# 7.3.

- Please refer... --> See \[Select the C# language version](../../../language-reference/configure-language-version.md).

Reply:
OK I will do the required changes